### PR TITLE
Fix next prev

### PR DESCRIPTION
**Pull request by mark-tate**
fixes next/prev page for @JoanaMMoreira 
removes snapshot api for @DavieReid

### CODE REVIEW

File: packages/site-components/src/DocPaginator/__tests__/DocPaginator.test.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import { render, screen } from '@testing-library/react';
+import userEvents from '@testing-library/user-event';
+import mockRouter from 'next-router-mock';
+import { MemoryRouterProvider } from 'next-router-mock/MemoryRouterProvider';
+import NextLink from 'next/link';
+import { LinkProvider } from '@jpmorganchase/mosaic-components';
+
+import { DocPaginator } from '../DocPaginator';
+
+describe.only('GIVEN a DocPaginator', () => {
```
Comment (DavieReid) [inline packages/site-components/src/DocPaginator/__tests__/DocPaginator.test.tsx:11]: `only` left in

File: packages/site-components/src/DocPaginator/__tests__/DocPaginator.test.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React, { PropsWithChildren } from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvents from '@testing-library/user-event';
+import mockRouter from 'next-router-mock';
```
Comment (DavieReid) [inline packages/site-components/src/DocPaginator/__tests__/DocPaginator.test.tsx:4]: nice

**Review by DavieReid — APPROVED** [review]

File: packages/site-components/src/DocPaginator/__tests__/DocPaginator.test.tsx (hunk 1 above)
Comment (mark-tate) [reply packages/site-components/src/DocPaginator/__tests__/DocPaginator.test.tsx]: removed

**Comment by mark-tate** [comment]
good shout @lilyvc it is not supported by the plugin yet but we can add... I'll follow up with another PR for that

**Pull request merged**